mne/viz/raw.py

```diff
@@ -566,31 +566,59 @@ def _set_psd_plot_params(info, proj, picks, ax, area_mode):
             ax_list, make_label)
 
 
-def _convert_psds(psds, dB, scaling, unit, ch_names):
-    """Convert PSDs to dB (if necessary) and appropriate units."""
-    if dB:
-        where = np.where(psds.min(1) <= 0)[0]
-        if len(where) > 0:
-            raise ValueError("Infinite value in PSD for channel(s) %s. "
-                             "These channels might be dead." %
-                             ', '.join(ch_names[ii] for ii in where))
+def _convert_psds(psds, dB, estimate, scaling, unit, ch_names):
+    """Convert PSDs to dB (if necessary) and appropriate units.
+
+    The following table summarizes the relationship between the value of
+    parameters `dB` and `estimate`, and the type of plot and corresponding
+    units.
+
+    | dB    | estimate    | plot | units           |
```

Comment:
This table would be easier to read if you structured it differently:

```
| dB    | estimate    | plot | units            |
|-------+-------------+------+------------------|
| True  | 'power'     | PSD  | uV**2/Hz (dB)    |
| True  | 'amplitude' | ASD  | uV/sqrt(Hz) (dB) |
| True  | 'auto'      | PSD  | uV**2/Hz (dB)    |
| False | 'power'     | PSD  | uV**2/Hz         |
| False | 'amplitude' | ASD  | uV/sqrt(Hz)      |
| False | 'auto'      | ASD  | uV/sqrt(Hz)      |
```

Please also use `uV` and not `u` as units. Finally, I'd write `sqrt(Hz)` and not `sqrt{Hz}`.

Reply:
OK with the table order. Units are not uV though. I'm using u as a general unit, since this can be MEG, etc. I'll change it to 'amp', as in amplitude to avoid the confusion.